sqrl-engines/sqrl-engine-server/src/main/java/com/datasqrl/engine/server/LambdaJavaEngineFactory.java

```diff
@@ -0,0 +1,31 @@
+package com.datasqrl.engine.server;
+
+import com.datasqrl.config.SqrlConfig;
+import com.datasqrl.engine.EngineFactory;
+import com.datasqrl.engine.ExecutionEngine;
+import com.datasqrl.engine.server.LambdaNativeArm64EngineFactory.LambdaNativeArm64Engine;
+import com.google.auto.service.AutoService;
+import lombok.NonNull;
+
+@AutoService(EngineFactory.class)
+public class LambdaJavaEngineFactory extends GraphqlServerEngineFactory {
+
+  public static final String ENGINE_NAME = "aws-lambda-native";
```

Comment:
You have an engine for arm and x86 - what is this generic native lambda engine for?

Reply:
Java. So if we put in calcite which does codegen, we likely wouldn't be able to use native compilation there.